pkg/operator/prober_test.go

```diff
@@ -66,15 +68,15 @@ func TestProbers(t *testing.T) {
 				if tc.err {
 					if err == nil {
 						t.Logf("%s: %s", strings.Join(args, " "), string(b))
-						t.Fatal("expecting error but got nil")
 					}
+					require.Error(t, err, "expecting error but got nil")
```

Comment:
Nit

```suggestion
					require.Error(t, err)
```

The message before was useful because t.Fatal by itself doesn't add any context to the failure. This changes when we use `require.Error`, making the extra message unnecessary :)